website/index.html

```diff
@@ -29,9 +29,27 @@
         <div class="large-12 columns">
           <div class="top-section border-all">
             <img src="images/rh-labs-logo.svg" class="rh-logo float-right" />
-            <h1>Jump start your development with a custom built environment</h1>
-            <h3>Design your own Residency-ready technology stack.</h3>
-            <p>Use this interactive menu to mix and match the services, tools, and infrastructure to create your own, customized, Open Innovation Lab technology stack that will help to accelerate your application development. Click submit to review your own personalized technology stack, which you can save, share, and use as a springboard for your own app development.</p>
+            <h1>
+              Let us Explore our Technology Together.
+            </h1>
+            <h3>
+              Design your own engagement-ready technology stack
+            </h3>
+            <hr />
+            <p>
+              Use the interactive menu below to create your own, customized tech stack. Accelerate your move to modern technologies and practices.
```

Comment:
Please update to the doc text:
"Use the interactive menu below to create your own, customized, technology stack to accelerate your move to modern technologies and practices."